openfisca_core/scripts/__init__.py

```diff
@@ -28,7 +29,10 @@ def build_tax_benefit_sytem(country_package_name, extensions, reforms):
     try:
         country_package = importlib.import_module(country_package_name)
     except ImportError:
-        handle_error(u'Could not import module `{}`. Make sure it is installed in your environment.'.format(country_package_name))
+        handle_error(u'{}\nCould not import module `{}`.\n'.format(traceback.format_exc(), country_package_name)
```

Comment:
`\n` would not work on Windows.
To make your code cross-platfrom, you can use [`os.linesep`](https://docs.python.org/3/library/os.html#os.linesep) instead.

> You may want to use the `join` string method:
>```py
>message = os.linesep.join([
>    "First line",
>    "Second line"
>    ])
>print(message)
>>>> First line
>>>> Second line
>```